src/main/java/com/tabgok/heimdallr/Heimdallr.java

```diff
@@ -0,0 +1,22 @@
+package com.tabgok.heimdallr;
+
+import com.tabgok.entity.factory.MachineFactory;
+import com.tabgok.server.Gateway;
+
+
+public class Heimdallr {
+    /**
+     * Starts up an instance of the Heimdallr system, which consists of two parts:
+     *  1. Start a machine factory, which gathers all the information on the system and constructs an in-memory view of the local "Machine".
+     *  2. Start up a "Gateway", which is an HttpServer, in order to respond to queries and provide JSON data
+     * @param args No arguments expected
+     */
+    public static void main(String args[]){
+        //Create the factory to create an in-memory "Machine" 
+        MachineFactory factory = new MachineFactory();
```

Comment:
ctors => factory?

I've read conflicting things on singletons and static classes, if MachineFactory.create() makes sense, I think I'd want it to create a "Machine" instead of a "MachineFactory".  Also, I think I'd make MachineFactory() private and provide all the functions as static.